_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,152 @@
+---
+layout:       element
+title:        Forms
+date:         2015-05-12 16:02:35
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- forms
+
+slug:         forms
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/edx-pl/sass/components/_forms.scss"
+
+description:  Various examples of different form fields and their usage and options.
+
+info: Forms allow users to interact with the interface, set preferences, and enter any necessary personal information. They also form the base of individual exercises within each class.
+---
+
+<h3 class="hd-4 example-set-hd">Forms</h3>
+<h4 class="hd-5">Text inputs</h4>
+<div class="example-set">
+    <h5 class="hd-6">Default</h5>
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Personal Details</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="full_name_default">Your name</label>
+            <input class="input input-text" type="text" name="full_name_default" id="full_name_default" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="email_address_default">Email address</label>
+            <input class="input input-text" type="email" name="email_address_default" id="email_address_default" placeholder="e.g. jane@email.com" disabled>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="email_address_default_helper">Email address</label>
+            <input class="input input-text" type="email" name="email_address_default_helper" id="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper" required>
+            <div class="helper-text" id="email_address_default_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label" for="full_name_simple">Your name</label>
+            <input class="input input-text input-simple" type="text" name="full_name_simple" id="full_name_simple" placeholder="e.g. Jane Doe" required>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-5">Select/Dropdown/Combo</h4>
+<div class="example-set">
+    <fieldset class="fieldset">
+        <legend class="group-legend is-hidden">Some select menus</legend>
+        <div class="field-wrapper">
+            <label class="form-label" for="select_01">Select something already!</label>
+            <select class="input input-select replace-select" name="select_01" id="select_01">
+                <option>- Choose one -</option>
+                <option value="Mary">Mary</option>
```

Comment:
Since I added `value=""` above I think we should keep it here for clarity and consistency. Programmers would likely know when or when not to use certain aspects.